doc/specification.md

```diff
@@ -170,6 +170,19 @@ Implications:
     * Get() or Flush() should return failed (because the VRF is no longer there)
     * When the VRF is added back, the server is not required to restore all the gRIBI objects by itself.
 
+### 4.1.10 Coalesced AFTOperations
+
+In some scenarios, a device might coalesce multiple AFTOperations on a given gRIBI entry and only execute the last one. This would be primarily done for performance optimization.
+
+In this case, as long as the session is still up and the client is still the primary client, the device SHOULD ACK/NACK (defined in x.y.z) each individual AFTOperation from the same primary client.
+
+The reasonings are:
+* Keep the API behavior clear and consistent.
+* The device should already have context of all the pending AFTOperations, ACK/NACK multiple coalesced AFTOperations based on one executed should not be a huge cost.
+* Does not require the sender to track the content of the pending AFTOperations.
+
```

Comment:
```suggestion

The server (device) has context of all pending `AFTOperation` messages, since it must potentially ACK any individual operation. Sending an ACK/NACK per message does not present a significant cost.
```